apps/earn-protocol/features/transak/components/TransakExchange/TransakExchange.tsx

```
@@ -1,9 +1,9 @@
 import { type Dispatch, type FC, useEffect, useState } from 'react'
 import { Icon, Text, Tooltip, useMobileCheck } from '@summerfi/app-earn-ui'
+import { REVALIDATION_TIMES } from '@summerfi/app-earn-ui'
```

ESLint

[error] '@summerfi/app-earn-ui' import is duplicated.

(no-duplicate-imports)